scripts/apps/authoring-react/data-layer.ts

```diff
@@ -122,6 +123,28 @@ function getArticleContentProfile<T>(item: IArticle, fieldsAdapter: IFieldsAdapt
             }
         }
 
+        // TODO: write an upgrade script and remove hardcoding
+        // after angular based authoring is removed from the codebase
+        if (item.type === 'picture') {
+            const fieldConfig: IEditor3Config = {
+                editorFormat: [],
+                minLength: fakeScope.schema?.minlength,
+                maxLength: fakeScope.schema?.maxlength,
+                cleanPastedHtml: fakeScope.editor?.cleanPastedHTML,
+                singleLine: true,
```

Comment:
is that single line in angular version?

Reply:
I can't check it in sd-develop now, so I don't know.

Reply:
why can't you check on sd-develop? Would it work to check locally then with angular based authoring?

Reply:
Checked it on the local instance, we do allow multi line.